test/jit/test_schema_check.py

```diff
@@ -96,3 +142,35 @@ def test_mutation_check_fail_multiple_operators(self):
                 x.tanh_()
                 x.relu_()
                 batch(SchemaCheckMode(x))
+
+    # Tests that an exception is raised for a mismatching alias
+    def test_alias_check_fail(self):
+        with self.assertRaises(RuntimeError):
```

Comment:
nit: for best practice, can you use `self.assertRaisesRegex(RuntimeError, "my expected error message")`?